src/app/modules/project/overview/overview.component.ts

```diff
@@ -82,7 +81,7 @@ export class OverviewComponent implements OnInit {
    */
   public defaultPaginationOption = {
     id: 0,
-    amountOnPage: 10
+    amountOnPage: 10,
```

Comment:
```suggestion
    amountOnPage: 10
```

Reply:
With the comma is better in my opinion.
If a new property gets added to the field, only one line is added instead of also having to add a comma to the property above.

Reply:
I am like Switzerland when it comes to these kind of things